main.rb

```diff
@@ -20,7 +21,7 @@ def inputs(int)
     end
```

Comment:
- [ ] Kindly remove the `starting` word at line 20 in the `inputs` method as it causing an error.